packages/ckeditor5-engine/src/model/differ.ts

```diff
@@ -1415,21 +1486,17 @@ export interface DiffItemInsert {
 	type: 'insert';
 
 	/**
-	 * Reference to the model element that was inserted.
-	 *
-	 * Undefined if the diff item is related to text node insertion.
-	 *
-	 * @internal
+	 * Indicates if insert is part of operation that consist multiple suboperations.
```

Comment:
> I still believe it would be easier to document if action had 'insert' and 'remove' values ;). We would avoid all "Not sets" and move the second remark to ChangeItemAction docs.

From what I can see, it's not trivial modification and marking `ChangeItem.action` attribute required involves us in adding new action for attributes change and besides that we have to do adjustments in all engine methods due to typing issues. The current approach is much simpler in terms of affected types, code and not introduce any breaking change in `DiffItemInsert` / `DiffItemRemove` typings.

Reply:
:+1: